package/MDAnalysis/analysis/contacts.py

```diff
@@ -338,6 +338,28 @@ def contact_matrix(d, radius, out=None):
     return out
 
 
+def get_box(ts, pbc):
+    """Retrieve the dimensions of the simulation box based on PBC.
```

Comment:
This function is very general, so it should be in a core module I think. Otherwise it could be a lambda defined below (see comment).